src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "CSS backdrop-filter"
+subhead: |
+  Enabled in Chrome 76
+date: 2019-06-25
+authors:
+  - adamargyle
+hero: hero.jpeg
+alt: Women behind frosted glass
+description: |
+  CSS backdrop-filter was made famous by Apple's design team. It brings
+  the ability for a background to apply a filter to the contents beneath.
+  Creates a nice layering effect that keeps beneath elements in context.
+tags:
+  - post
+  - CSS
+  - backdrop-filter
+---
+
+Designers and developers alike can celebrate as a long awaited CSS feature [releases in Chrome 76](https://www.chromestatus.com/feature/5679432723333120): backdrop-filter. Frosted glass effects, video overlays, translucent navigation headers, inappropriate image censoring, image loading, and more, no longer require [hacks or workarounds](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter).
+
+You may recognize the effect from two popular operating systems: [Windows Vista](https://i.kinja-img.com/gawker-media/image/upload/s--9RLXARU4--/c_scale,dpr_2.0,f_auto,fl_progressive,q_80,w_800/trgz8yivyyqrpcnwscu5.png) and [iOS](https://static.businessinsider.com/image/51fd2822eab8eae16e00000b-750.jpg). Now [Chrome](https://www.chromestatus.com/feature/5679432723333120) can do it too.
+
+The following video demonstrates `backdrop-filter` and each individual filter:
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-kitchen_sink.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    <a href="https://codepen.io/robinrendle/pen/LmzLEL" target="_blank">CodePen</a>
```

Comment:
i left them all as simple links since all the data was there at the destination if they wanted it. like, what does original codepen mean, we're not showing a codepen? it does feel more courteous to include the authors name in the caption. maybe we go with "[CodePen](#) by Robin Rendle"?

Reply:
Sure that works